src/lib/state/swap.ts

```diff
@@ -27,3 +28,8 @@ export const independentFieldAtom = pickAtom(swapAtom, 'independentField')
 
 // If set to a transaction hash, that transaction will display in a status dialog.
 export const displayTxHashAtom = atom<string | undefined>(undefined)
+
+export const feeOptionsAtom = atom<FeeOptions>({
+  fee: new Percent(0),
```

Comment:
Define this as a default so it can be re-used in `useSwapInfo` (see comment in that file).